### PR TITLE
fix(cli): build command `features` and `bundles` can have multiple values

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: tooling/cli/src/build.rs [hunk 1]
```diff
@@ -46,15 +46,15 @@ pub struct Options {
...
+  #[clap(short, long, action = ArgAction::Append, num_args(0..))]
   pub features: Option<Vec<String>>,
   /// Space or comma separated list of bundles to package.
   ///
   /// Each bundle must be one of `deb`, `rpm`, `appimage`, `msi`, `app` or `dmg` on MacOS and `updater` on all platforms.
   /// If `none` is specified, the bundler will be skipped.
   ///
   /// Note that the `updater` bundle is not automatically added so you must specify it if the updater is enabled.
-  #[clap(short, long, value_delimiter = ',')]
+  #[clap(short, long, action = ArgAction::Append, num_args(0..))]
```
Comment (amrbashir) [inline tooling/cli/src/build.rs]: Not sure what this solves tbh, `Option<Vec<T>>` will use `ArgAction::Append` and `num_args(0..)`, This is kinda redundant. Also not using value_delimiter, means we need to handle it ourselves which I removed in favor of clap built-in value delimiter

Comment (lucasfernog) [reply tooling/cli/src/build.rs]: that's not really true, just try it yourself, `tauri build --features a b` won't work

Comment (lucasfernog) [reply tooling/cli/src/build.rs]: plus there's more args using this

Comment (amrbashir) [reply tooling/cli/src/build.rs]: Ah maybe clap docs is not clear about the num_args thing but they do use append by default `--features a,b,g --features s` should work

Comment (amrbashir) [reply tooling/cli/src/build.rs]: Anyways, if we go with manual delimiter, we need to add the code to handle it

**Review by amrbashir — APPROVED** [review]

**Pull request merged**